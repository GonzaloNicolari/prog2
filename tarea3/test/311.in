# prueba de suma_ultimos_pares
suma_ultimos_pares 0
suma_ultimos_pares 1
insertar_en_binario (2,g)
suma_ultimos_pares 0
suma_ultimos_pares 1
suma_ultimos_pares 2
insertar_en_binario (1,m)
suma_ultimos_pares 0
suma_ultimos_pares 1
suma_ultimos_pares 2
insertar_en_binario (4,s)
suma_ultimos_pares 0
suma_ultimos_pares 1
suma_ultimos_pares 2
insertar_en_binario (1,i)
insertar_en_binario (-2,j)
insertar_en_binario (1,k)
insertar_en_binario (10,h)
suma_ultimos_pares 2
suma_ultimos_pares 3
suma_ultimos_pares 4
insertar_en_binario (3,d)
suma_ultimos_pares 5
insertar_en_binario (6,f)
suma_ultimos_pares 5
Fin