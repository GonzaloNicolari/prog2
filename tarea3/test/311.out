1>#  prueba de suma_ultimos_pares.
2>0
3>0
4>Insertado.
5>0
6>2
7>2
8>Insertado.
9>0
10>2
11>2
12>Insertado.
13>0
14>4
15>6
16>Insertado.
17>Insertado.
18>Insertado.
19>Insertado.
20>2
21>12
22>14
23>Insertado.
24>14
25>Insertado.
26>20
27>Fin.
